app/js/lib/intent-parser/confirmation.js

```diff
@@ -0,0 +1,151 @@
+/* global TwitterCldr, TwitterCldrDataBundle */
+
+'use strict';
+
+import moment from 'components/moment';
+import 'components/cldr/en';
+import 'components/cldr/core';
+
+/*
+ * @todo:
+ *   * @see http://www.unicode.org/cldr/charts/29/verify/dates/en.html
+ *     for formatting the time of the day.
+ */
+
+const p = Object.freeze({
+  // Properties
+  listFormatter: Symbol('listFormatter'),
+
+  // Methods
+  formatUser: Symbol('formatUser'),
+  formatAction: Symbol('formatAction'),
+  formatTime: Symbol('formatTime'),
+  isToday: Symbol('isToday'),
+  isTomorrow: Symbol('isTomorrow'),
+  isThisMonth: Symbol('isThisMonth'),
+  formatHoursAndMinutes: Symbol('formatHoursAndMinutes'),
+});
+
+const PATTERNS = {
+  en: {
+    pattern: `OK, I'll remind [users] [action] [time].`,
+  },
+  fr: {
+    pattern: `OK, je rappelerai [users] [action] [time].`,
+  },
+  ja: {
+    pattern: `承知しました。[time][users]に[action]をリマインドします。`,
+  },
+};
+
+export default class Confirmation {
+  constructor(locale = 'en') {
+    this.locale = locale;
+
+    TwitterCldr.set_data(TwitterCldrDataBundle);
+
+    this[p.listFormatter] = new TwitterCldr.ListFormatter();
+  }
+
+  /**
+   * Generate a phrase to be spoken to confirm a reminder.
+   *
+   * @param {Object} reminder
+   * @return {string}
+   */
+  getReminderMessage(reminder) {
+    console.log(reminder);
+
+    const pattern = PATTERNS[this.locale].pattern;
+    const data = {
+      users: this[p.formatUser](reminder),
+      action: this[p.formatAction](reminder),
+      time: this[p.formatTime](reminder),
+    };
+
+    return pattern.replace(/\[([^\]]+)\]/g, (match, placeholder) => {
+      return data[placeholder];
+    });
+  }
+
+  [p.formatUser](reminder) {
+    const users = reminder.users
+      .map((user) => user.toLowerCase() === 'me' ? 'you' : user);
```

Comment:
`this` object has a 'locale' property.  I realise l10n is a huge task, but it would be awesome if we could keep it "localisable" at least.  Do you think it's worth pulling these strings out?